fdbserver/DDTeamCollection.actor.cpp

```diff
@@ -3253,6 +3226,36 @@ class DDTeamCollectionImpl {
 	}
 }; // class DDTeamCollectionImpl
 
+void DDTeamCollection::updatePivotAvailableSpaceRatio() {
+	if (now() - lastPivotAvailableSpaceUpdate > SERVER_KNOBS->AVAILABLE_SPACE_UPDATE_DELAY) {
+		lastPivotAvailableSpaceUpdate = now();
+		std::vector<double> teamAvailableSpace;
+		teamAvailableSpace.reserve(teams.size());
+		for (const auto& team : teams) {
+			if (team->isHealthy()) {
+				teamAvailableSpace.push_back(team->getMinAvailableSpaceRatio());
+			}
+		}
+
+		size_t pivot = teamAvailableSpace.size() * std::min(1.0, SERVER_KNOBS->AVAILABLE_SPACE_PIVOT_PERCENT);
```

Comment:
This line was `size_t pivot = teamAvailableSpace.size() / 2;` in the old code, so the default value for `AVAILABLE_SPACE_PIVOT_PERCENT` should be 0.5 rather than 0.6 `ServerKnobs.cpp` to prevent behavior change.